### PR TITLE
Fixed BOSH over websocket.

### PR DESCRIPTION
**Pull request by sonnguyen1205**
Fixed problem when using BOSH over websocket with notice in console log "status changed: connecting".

### CODE REVIEW

File: src/JSJaCConstants.js [hunk 1]
```diff
@@ -40,6 +40,7 @@ var NS_PUBSUB_NMI =   "http://jabber.org/protocol/pubsub#node-meta-info";
 var NS_COMMANDS =     "http://jabber.org/protocol/commands";
 var NS_CAPS =         "http://jabber.org/protocol/caps";
 var NS_STREAM =       "http://etherx.jabber.org/streams";
+var NS_FRAMING =      "urn:ietf:params:xml:ns:xmpp-framing";
```
Comment (sstrigler) [inline src/JSJaCConstants.js]: Apparently this is unused within JSJaC, so please remove.

File: src/JSJaCWebSocketConnection.js [hunk 2]
```diff
@@ -346,11 +346,8 @@ JSJaCWebSocketConnection.prototype._getInitialRequestString = function() {
     streamto = this.authhost;
   }
 
-  reqstr = '<stream:stream to="' + streamto + '" xmlns="jabber:client" xmlns:stream="' + NS_STREAM + '"';
-  if (this.authtype === 'sasl' || this.authtype === 'saslanon') {
```
Comment (sstrigler) [inline src/JSJaCWebSocketConnection.js]: Sure this is not needed or possible anymore? Haven't read the specs though ...

**Comment by sstrigler** [comment]
Accepting this the way it is (without a flag) will break with outdated implementations. I think that would be ok.

**Comment by sstrigler** [comment]
Oh, and thanks a lot for that nice contribution! 👍

**Comment by amishxda** [comment]
Corrected above patch further with proper XML calls.

https://github.com/sstrigler/JSJaC/pull/88

**Pull request closed without merging**